src/main/java/de/tum/in/www1/artemis/security/jwt/JWTFilter.java

```diff
@@ -51,7 +51,7 @@ public static ResponseCookie buildJWTCookie(String jwt, Duration duration) {
 
         return ResponseCookie.from(JWT_COOKIE_NAME, jwt).httpOnly(true) // Must be httpOnly
                 .sameSite("Lax") // Must be Lax to allow navigation links to Artemis to work
-                .secure(true) // Must be secure
+                .secure(false) // Must be secure
```

Comment:
See your own comment :)

Reply:
Eheh I was (again) testing if the cypress tests fail only when the cookie is secure, and indeed they do. We're figuring out a solution for that, which will probably be using https for the cypress tests as well